addons/web/static/tests/views/list_tests.js

```diff
@@ -1052,6 +1052,55 @@ QUnit.module('Views', {
         actionManager.destroy();
     });
 
+    QUnit.test('list view without groupby menu', async function (assert) {
+        assert.expect(2);
+
+        var searchMenuTypesOriginal = ListView.prototype.searchMenuTypes;
+        ListView.prototype.searchMenuTypes = ['filter', 'favorite'];
+
+        var actionManager = await createActionManager({
```

Comment:
Same remark concerning createActionManager. We should prefer createView whenever it is possible

Reply:
@Polymorphe57 Same remark as above ;)